assets/css/creative.scss

```diff
@@ -34,12 +34,12 @@
   --danger: #dc3545;
   --light: #f8f9fa;
   --dark: #343a40;
-  --saibrotech-orange-dark: #ff3800;
-  --saibrotech-orange: #ff6600;
-  --saibrotech-orange-light: #ff7f00;
-  --saibrotech-blue-dark: #0086ff;
-  --saibrotech-blue: #00b6ff;
-  --saibrotech-blue-light: #00e1ff;
+  --saibrotech-orange-dark: rgba(255, 56, 0, 0.9);
+  --saibrotech-orange: rgba(255, 102, 0, 0.9);
+  --saibrotech-orange-light: rgba(255, 127, 0, 0.9);
+  --saibrotech-blue-dark: rgba(0, 134, 255, 0.9);
+  --saibrotech-blue: rgba(0, 182, 255, 0.9);
+  --saibrotech-blue-light: rgba(0, 225, 255, 0.9);
```

Comment:
- changed HEX colors to rgba with 0.9 alpha, to address low contrast issues;
- using the defined color variables whenever possible throughout the code;